alf/algorithms/mi_estimator.py

```diff
@@ -121,7 +122,7 @@ def __init__(self,
                 'shuffle', 'shift']
             buffer_size (int): capacity of buffer for storing y for sampler
                 'buffer' and 'double_buffer'
-            optimzer (tf.optimizers.Optimzer): optimizer
+            optimzer (torch.optim.Optimzer): optimizer
```

Comment:
update the docstring and remove "Create a ..." because this docstring will be combined with the class docstring above.

Reply:
doen